tsconfig.json

```
@@ -12,11 +12,13 @@
     "strict": true,
     "noImplicitAny": false,
     "outDir": "./dist",
+    "rootDir": "./src",
```

Maybe make it just the root to keep structure?
```suggestion
    "rootDir": ".",
```

---

The the output will be `dist/src/...` instead of `dist/...`